src/main/java/core/basesyntax/DaySummary.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class DaySummary {
+    private final String date;
+    private final String name;
+    private final int hours;
+    private final int incomePerHour;
+    private final int daySumIncome;
+
+    public DaySummary(String data) {
+        this.date = data.split(" ")[0];
+        this.name = data.split(" ")[1];
+        this.hours = Integer.valueOf(data.split(" ")[2]);
+        this.incomePerHour = Integer.valueOf(data.split(" ")[3]);
+        this.daySumIncome = this.hours * this.incomePerHour;
+    }
```

Comment:
I think It's better to parce String to DaySummary in some service (or just in some private method of SalaryInfo class) but not in constructor